app/backend/src/couchers/servicers/gis.py

```diff
@@ -37,7 +37,7 @@ def _query_to_geojson_response(session, query):
 class GIS(gis_pb2_grpc.GISServicer):
     def GetUsers(self, request, context):
         with session_scope() as session:
-            query = session.query(User.username, User.id, User.geom).filter(User.geom != None)
+            query = session.query(User.username, User.id, User.geom).filter(User.is_visible).filter(User.geom != None)
```

Comment:
Is GIS tested anywhere? I just realized I never tested this change.

Reply:
No, these endpoints are not tested!